### ISSUE TITLE
Hundreds of weird Top-Level rem show in my Document table

### ISSUE THREAD

**Issue by ognsya** [remnoteio/remnote-issues#713]
**Describe the bug**
If I go to my Document table, it shows hundreds of top-level rem that I did not create. Most are numbers (see image below), but there are also some that are single characters. The Rems themselves are empty (see image). Not sure when/why these exist. I only use a local-only KB. Never did import/export from anywhere.

As far as I know, this doesn't really affect my use of the app, except for making it very difficult to find (actual) toplevel Rems.

**To Reproduce**
Not sure, as I don't know how these Rem appeared there ( and why they are numbers).

**Expected behavior**
No mistery Rems cluttering my KB

**Screenshots**
See below

**Desktop (please complete the following information):**
Windows
Desktop
v 1.1.10

![image](https://user-images.githubusercontent.com/67518000/106744875-3a66dc80-6618-11eb-87bc-603f51285590.png)

![image](https://user-images.githubusercontent.com/67518000/106744907-46eb3500-6618-11eb-96a6-df8a1f6add9d.png)

**Comment by mfranzs** [comment]
Thanks for reaching out. 

1. Are any of these Rem linked or referenced anywhere else? (Ex. marked as a tag?)
2. Were they all created on Nov 29th? If so, do you know what the significance of that date might be? (Ex. did you create the knowledge-base then, did you paste something, etc)
3. Is it possible you pasted something which had these as tags?
4. Are they tagged as "Stub"?

**Comment by ognsya** [comment]
TLDR: I managed to reproduce the issue! It happens when you paste a full wikipedia page into a Document in RemNote (which generates hundreds of Rems), and then delete all those Rems. Most Rems are indeed deleted (as expected), but some specific ones aren't (possibly because they were read as tags when originally pasted in).

**Here are the steps**
**!Warning: do it in a test KB!**
Step 1: Go to [some wikipedia page](https://en.wikipedia.org/wiki/United_States) and copy everything (CTRL+A, CTRL+C)
Step 2: Create a new Rem and paste into it (might take a minute)
Step 3: Delete all those newly created Rems (CTRL+A, DEL)
Step 4: Go to Documents and verify that all content from the wikipedia page is gone except for some specific Rems. From what I saw, this is: all numbers referring to the footnotes (in the case of the article linked above, it goes up to 508), all letters from a-z and the word "show". (I checked the html for the wikipedia page and it seems all footnote numbers are mentioned in the format `<a href="#cite_ref-508">`, so maybe the hashtag generated tags as you suggested; though the "cite_ref" part doesn't seem to persist) 

//

**Answering your questions**: (though you can probably check it for yourself based on the above)
1. No. All of them are empty expect for the content of the Rem, ie. the "name" of the Document itself (nothing at the bottom saying it's referenced or tagged)
2. Good call. I did realize they all had the same date but forgot to check it's significance. The date is from when I first started using the app. I probably did a test by pasting a full wikipedia page into a Rem to test performance (which indeed generates these stray top-level Rems).
3. Possibly (though not intentionally), as mentioned in Step 4 above.
4. No.

Glad to know this was just a freak specific event.
Now I just need to figure out a sane way to delete all those stray Rems!

**Comment by ognsya** [comment]
Two more bits of info:
- I'm experiencing another issue, which is that the tags of the currently open Rem sometimes do not show on top. I couldn't figure out what's the logic on this. 
- I realized that if I do the wikipedia test above now, the stray Rems **do** have the #Stub tag. But the ones I created back in Nov do not (maybe it was a change in how RemNote imports?). To check this I'm looking at the Stub page's references.

**Comment by mfranzs** [comment]
Great, thanks! We'll make sure these aren't copied as tags.

**Comment by ognsya** [comment]
> Great, thanks! We'll make sure these aren't copied as tags.

Thanks.
I can't find a practical way to delete these hundreds of stray Rem (as they aren't tagged).
The only way I found was to go one by one in the Document list, going in and deleting.
Is there some better way?

Maybe some way to list all top-level documents that are not tagged, in a way that I can then select them and delete at once?

**Comment by mfranzs** [comment]
Here's a really quick script I found that should hopefully find them: https://www.remnote.io/run_support_ticket/601b2f0301f5980042b4fa7f

In the future, this should be better supported through our query language.

**Comment by ognsya** [comment]
How do I run that in desktop, on my local KB?
